src/wattsi.pas

```diff
@@ -850,20 +850,24 @@    TCrossReferences = record
             end;
          end
          else
-         if ((Element.IsIdentity(nsHTML, eSpan)) and (Element.GetAttribute('class').AsString = 'pubdate')) then
+         if ((Element.IsIdentity(nsHTML, eSpan)) and ((Element.GetAttribute('class').AsString = 'pubdate') or (Element.GetAttribute('class').AsString = 'pubyear'))) then
          begin
+            ClassName := Element.GetAttribute('class').AsString;
```

Comment:
Hoisting this up to inside a `if (Element.IsIdentity(nsHTML, eSpan)` and indenting a bit would be less duplication.